scripts/globals/items/koga_shuriken_pouch.lua

```diff
@@ -0,0 +1,25 @@
+-----------------------------------------
+-- ID: 6300
+-- Item: Koga Shr. Pouch
+-- When used, you will obtain one stack of Koga Shurikens
+-----------------------------------------
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0;
+    if (target:getFreeSlotsCount() == 0) then
+        result = 308;
+    end
+return result;
```

Comment:
missing indent here

Reply:
@TeoTwawki - used a shell of another item that didn't have the indent. Will be more mindful in the future as I work more :)

Reply:
Been fixing old files indention and other messes as I locate/see them so that people don't copy :) happens a lot so don't feel bad about it